doc/source/plans.rst

```diff
@@ -219,6 +221,11 @@ incorporating these trajectories, use our general N-dimensional scan plan,
    grid_scan
    rel_grid_scan
    scan_nd
+   list_scan
```

Comment:
Should the `list_scan` plan be removed from the previous block?

Reply:
I think they belong in both, as they now scan over 1 _or_ more dimensions so are both 1 dimensional scans _and_ multi-dimensional scans.

Reply:
OK, sounds good.